bprof-spec-codes/workcal
# Create a docker image with data volume for the application backend layer

To test out the solution there are a few requirements:
- WSL and hyper-v enabled.
- Docker desktop application installed on the machine. https://docs.docker.com/get-docker/
- At least 4 GB memory

Test plan:
- Pull the branch.
- Change the directory in a terminal to the project root (where the docker-compose.yml file is located.)
- "docker-compose up -d" (FYI: the -d flag just making not to run in the terminal where the command executed).
- Start the application from visual studio or Rider.
- You should see after application is ready the swagger, and also can use all endpoint.
- Also please check if shutdown the docker container and restart it, every data still in the project and in the database!
